File: Factorial/Java/ghostsnstuff/Factorial_Test.java
public class Factorial_Test {
  
  public static void main(String[] args) {
    System.out.println("The anticipated output for 5! = 120"); 
    Factorial factorial = new Factorial();
    int result = factorial.computeFactorial(5);
    System.out.print("factorial.computeFactorial(5) = ");
    System.out.println(result);
  }

}

File: Factorial/Scala/aayushKumarJarvis/FactorialTest.scala
import org.scalatest.{Matchers, FunSuite}

class FactorialTest extends FunSuite with Matchers {

  test("Factorial should return") {
    val objectForFactorial = new Factorial

    objectForFactorial.computeFactorial(2) should be("2")
    objectForFactorial.computeFactorial(5) should be("120")
    objectForFactorial.computeFactorial(10) should be("3628800")
    objectForFactorial.computeFactorial(50) should be("30414093201713378043612608166064768844377641568960512000000000000")
  }
}
